### DISCUSSION TITLE
Manual Install Instructions

### DISCUSSION THREAD

**Discussion by dave0003** [karakeep-app/karakeep#115] [General]
Do you have manual install instructions somewhere, that don't use docker?

**Comment by MohamedBassem** [comment]
Hi, it's definitely possible to install it without docker but to be honest I think it'll be quite painful. You'll have to install the dependencies manually (e.g. redis and meilisearch), and you'll need to come up with ways to daemonize the web and workers servers (e.g. with systemd). I'm also currently not publishing the binaries to npm, so you'll kinda need to build it from source. And on top of that, you'll be managing the updates manually. I'd be happy to help you get there, but it's highly not recommended :D In which environment do you need to run it without docker? LXC containers?

**Comment by dave0003** [comment]
I'm on debian.  I just prefer not to use docker.  If you have instructions written up already, I could take a look at them, but if not, it's ok.  No need to do anything special just for me.  Thanks.
